src/console.c

```diff
@@ -103,3 +103,18 @@ void console_init()
 	console_reset();
 	console_putstring("\nconsole: initialized\n");
 }
+
+void console_set_fgcolor(int r, int g, int b)
+{
+    fgcolor.r = r % 256;
```

Comment:
Why are we modding the values? Would it make more sense to simply round them to 256, so that setting a color of like {500,500,500} would still just be white? Otherwise this behavior becomes a bit harder to predict.